src/en/authors-hook-environment.md

```diff
@@ -479,3 +483,89 @@ relation-ids reverseproxy
 Note again that all commands that produce output accept `--format json` and
 `--format yaml`, and you may consider it smarter to use those for clarity's 
 sake than to depend on the default `smart` format.
+
+
+### status-set
+
+Introduced in version 1.24 of Juju, a new status mechanism allows Juju and its 
+charms to more accurately reflect their current status. This places the 
+responsibility on the charm to know its status, and set it accordingly using
+the `status-set` hook tool.
```

Comment:
the more detailed status will be exposed to anything that checks status, so yes, it does include the command-line tool. I don't think that really needs to be spelled out here, and it is more obvious from the following paragraphs anyhow